hooks/pre_build

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright 2018 The OpenSDS Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+BASE_DIR=`pwd`
+pwd
+apt-get update && apt-get install -y git make curl wget libltdl7 libseccomp2 libffi-dev gawk sudo
```

Comment:
There is no need to install these packages except make, curl and sudo, please remove them.